code/datums/components/shield.dm

```diff
@@ -168,20 +168,33 @@
 
 /datum/component/shield/proc/item_intercept_attack(attack_type, incoming_damage, damage_type, silent)
 	var/obj/item/parent_item = parent
+	var/status_cover_modifier = 1
+
+	if(affected.IsSleeping() || affected.IsUnconscious() || affected.IsAdminSleeping() || affected.IsParalyzed()) //We don't do jack if we're literally KOed/sleeping/paralyzed.
```

Comment:
This works so long as the `shield` in this case is something a mob has to hold in their hands. It probably doesn't work as well if the `shield` is some sort of all encompassing thing, like a belt shield or perhaps an armor module.

a check to see what the `parent` is in this case if possible please.

Reply:
The force shield works differently.